src/client/pages/OriginalDataPoint/components/ExtentOfForest/ExtentOfForestRow.tsx

```diff
@@ -133,7 +133,7 @@ const ExtentOfForestRow: React.FC<Props> = (props) => {
       </td>
 
       <td className="fra-table__calculated-cell">
-        {otherLand}
+        <span>{otherLand}</span>
```

Comment:
Added because when the value was 0.00, the table export would not pick it up. Wrapping it a span fixes it.